### PR TITLE
added note about Message Analyzer

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6815

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/client-management/troubleshoot-tcpip-netmon.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ manager: dansimp
 
 In this topic, you will learn how to use Microsoft Network Monitor 3.4, which is a tool for capturing network traffic.
 
+> [Note]
```
Comment (e0i) [inline windows/client-management/troubleshoot-tcpip-netmon.md:19]: @JohanFreelancer9 Is is the correct NOTE format? Thank you.

Comment (illfated) [reply windows/client-management/troubleshoot-tcpip-netmon.md:19]: The default MarkDown format for a Note blob should be like this:
```
> [!NOTE]
```
(although the word "NOTE" does not necessarily have to be in all caps)

Comment (illfated) [reply windows/client-management/troubleshoot-tcpip-netmon.md:19]:
```suggestion
> [!NOTE]
```

**Pull request merged**